cinema_service/settings.py

```diff
@@ -15,7 +15,6 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
```

Comment:
Make sure you've added a blank line at the end to all your files. It seems like the file is missing a blank line at the end.